src/locales/en.json

```
@@ -346,6 +346,7 @@
   "nft_portfolio_page_state_collections": "Collections",
   "nft_portfolio_page_state_collectors": "Collectors",
   "nft_portfolio_page_state_creations": "Creations",
+  "nft_portfolio_page_state_sales": "Total Revenue",

```

```suggestion
  "nft_portfolio_page_state_sales": "Total Revenue (LIKE)",

```
For consistency with zh version and asset value below